packages/app/obojobo-document-engine/src/scripts/oboeditor/util/validate-id.js

```diff
@@ -0,0 +1,6 @@
+const validateId = id => {
+	const validation = /[^a-z|A-Z|0-9|\-|_|:|.]/
+	return validation.test(id)
```

Comment:
I think this is a little confusing - `validateId` returns true if the given ID is *not* valid. Prefer to have this return true if the given id is valid.